elementor/elementor
# Fix: Update CSS specificity in order to override global color options [ED-15162]

## PR Checklist



## PR Type
What kind of change does this PR introduce?


## Summary

This PR can be summarized in the following changelog entry:

* Update CTA in Floating Button widget so it's not affected by global link colors

## Description
An explanation of what is done in this PR

* Update the CSS specificity for CTA link in Floating Button widget so it's not affected by global link colors

## Test instructions
This PR can be tested by following these steps:

* Update the global link color in Site settings
* Add a Floating Button variation that has a CTA link
* The CTA link color and hover color should not be affected by link colors in site settings

## Quality assurance


Fixes #
